src/backend/distributed/commands/owned.c

```diff
@@ -88,3 +88,36 @@ PreprocessDropOwnedStmt(Node *node, const char *queryString,
 
 	return NodeDDLTaskList(NON_COORDINATOR_NODES, commands);
 }
+
+
+List *
+PreprocessReassignOwnedStmt(Node *node, const char *queryString,
```

Comment:
Should we have this in Postprocess instead?

Reply:
There is no reason to move this to Postprocess actually. It just prepares the command. 
Normally when you check other statements who is in the nature of Alter like PreprocessAlterDatabaseStmt, we use Preprocess
Create needs to have Post since address only exists after local execution, Drop needs to be Pre since address is missing after local drop execution.
For Alter style statements, there is no such requirement.

Reply:
let's get this into postprocess as we've discussed in group chat, due to reasons discussed there

Reply:
> let's get this into postprocess as we've discussed in group chat, due to reasons discussed there

(discussion in group chat was still on-going indeed)